tests/zfs-tests/include/libtest.shlib

```diff
@@ -1972,6 +1973,13 @@ function check_pool_status # pool token keyword <verbose>
 #	is_pool_discarding - to check if the pool checkpoint is being discarded
 #	is_pool_replacing - to check if the pool is performing a replacement
 #
+
+function is_pool_without_errors #pool <verbose>
```

Comment:
Does this return true if there were no errors, or does it return true if there were errors but they were all repaired?

Reply:
Were there are no more known errors.